services/issue/issue.go

```diff
@@ -42,6 +42,20 @@ func ChangeTitle(issue *models.Issue, doer *models.User, title string) (err erro
 	return nil
 }
 
+// ChangeIssueRef changes the branch of this issue, as the given user.
+func ChangeIssueRef(issue *models.Issue, doer *models.User, ref string) (err error) {
+	oldRef := issue.Ref
+	issue.Ref = ref
+
+	if err = issue.ChangeRef(doer, oldRef); err != nil {
+		return
+	}
```

Comment:
the named return is not really helpful here.

```suggestion
func ChangeIssueRef(issue *models.Issue, doer *models.User, ref string) error {
	oldRef := issue.Ref
	issue.Ref = ref

	if err := issue.ChangeRef(doer, oldRef); err != nil {
		return err
	}
```